octoprint_mrbeam/templates/initial_calibration.jinja2

```diff
@@ -41,7 +41,7 @@
 				</ul>
 				<ul class="pull-right">
 					<li><a href="http://www.mr-beam.org" target="_blank"><i class="icon-home"></i> {{ _('Homepage') }}</a></li>
-					<li><a href="https://github.com/mrbeam/OctoPrint/" target="_blank"><i class="icon-download"></i> {{ _('Sourcecode') }}</a></li>
+					<li><a href="https://github.com/mrbeam/OctoPrint/" target="_blank"><i class="icon-download"></i> {{ _('Source code') }}</a></li>
```

Comment:
please do not translate any of the strings on this page, it's for internal use only.
Please delete the whole footer-element, it's not needed here.

Reply:
remove every translatable string